lib/shelley/bench/Latency.hs

```diff
@@ -345,11 +364,31 @@ benchWithShelleyServer tracers action = do
 
   where
     withServer act = withSystemTempDir nullTracer "latency" $ \dir -> do
-        params <- singleNodeParams Error
-        let db = dir </> "wallets"
-        createDirectory db
-        withBFTNode nullTracer dir params $ \socketPath block0 (gp, vData) ->
-            void $ serveWallet
+            testPoolConfigs' <- poolConfigsFromEnv
+            let db = dir </> "wallets"
+            createDirectory db
+            withCluster
+                nullTracer
+                Error
+                testPoolConfigs'
+                dir
+                onByron
+                (afterFork dir)
+                (onClusterStart act dir)
+    onByron _ = pure ()
+    afterFork dir _ = do
+        let encodeAddr = T.unpack . encodeAddress @'Mainnet
+        let addresses = map (first encodeAddr) shelleyIntegrationTestFunds
+        sendFaucetFundsTo stdoutTextTracer dir addresses
+        let rewards = (,Coin $ fromIntegral oneMillionAda) <$>
+                concatMap genRewardAccounts mirMnemonics
+        moveInstantaneousRewardsTo stdoutTextTracer dir rewards
```

Comment:
The MIR certificates aren't useful in the latency benchmarks, only the standard faucets are needed.

Reply:
indeed, fixed in https://github.com/input-output-hk/cardano-wallet/pull/2046/commits/e527debc14af47fcb558e432b16b443388364d68